vnext/ReactUWP/Views/TouchEventHandler.cpp

```diff
@@ -8,6 +8,8 @@
 
 #include <Modules/NativeUIManager.h>
 #include <UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.Documents.h>
```

Comment:
is this line necessary? you should get the right types with `UI.Xaml.Documents.h`

Reply:
right, forgot to remove it